### DISCUSSION TITLE
Rendering of Lightning web components

### DISCUSSION THREAD

**Discussion by HummerDK** [storybookjs/storybook#25111] [Help]
### Summary

Since Storybook supports native web components, i was thinking it should be possible to get storybook to display LWC's.

I have created a basic of storybook using framework "web-components-vite". I reference my LWC component, but it does not display. The constructor of the components is executed.

Do anyone have an idea on how to make the setup work?

**Comment by jonniebigodes** [comment]
Hey @HummerDK thanks for reaching out to us with your question. We appreciate it 🙏 ! I'm not that knowledgeable of Lightning Web Components, but I was wondering if you could create a small reproduction so that it can be looked at and help you solve this issue you're having.

Looking forward to hearing from you.

Have a great day.

Stay safe

**Comment by break-stuff** [comment]
@HummerDK this should be completely doable. We use all kinds of web components in Storybook. Any examples or repros would help us understand the problem.
